.node-version

```diff
@@ -0,0 +1 @@
+14.21.2
```

Comment:
Should we bump this to node 16.x since that's the latest lts? Or should we do that in a separate pr?

Reply:
yeah, we can bump it in a separate PR and check with netlify deployment.